common4j/src/main/com/microsoft/identity/common/java/providers/oauth2/OAuth2Strategy.java

```diff
@@ -140,6 +142,7 @@ public OAuth2Strategy(GenericOAuth2Configuration config,
         // Suppressing unchecked warnings due to casting an object in reference of current class to the child class GenericOAuth2Strategy while calling method requestAuthorization()
         @SuppressWarnings(WarningType.unchecked_warning) final Future<AuthorizationResult> authorizationFuture =
                 authorizationStrategy.requestAuthorization(request, this);
+        Telemetry.emit(new UiShownEvent().putVisible("true"));
```

Comment:
We have a constant for `true`..something like this:

```suggestion
        Telemetry.emit(new UiShownEvent().putVisible(TelemetryEventStrings.Value.TRUE));
```